packages/connectivity/src/scp-cf/destination/destination.ts

```diff
@@ -380,7 +380,9 @@ const configMapping: Record<string, keyof Destination> = {
   certificates: 'certificates',
   KeyStoreLocation: 'keyStoreName',
   KeyStorePassword: 'keyStorePassword',
-  SystemUser: 'systemUser'
+  SystemUser: 'systemUser',
+  'x_user_token.jwks': 'jwks',
```

Comment:
This map gets bigger over the time, would it be fine to add some docs about the keys, especially the new ones?

Reply:
added commnet for the new ones